docs/src/pages/components/data-grid/localization/localization.md

```diff
@@ -7,13 +7,19 @@ components: DataGrid, XGrid
 
 <p class="description">The Date Grid will support users from different locale, with formating, RTL, and localized strings.</p>
 
-## 🚧 Locale text
+The default locale of Material-UI is English (United States). If you want to use other locales, follow the instructions below.
 
-> ⚠️ This feature isn't implemented yet. It's coming.
->
-> 👍 Upvote [issue #196](https://github.com/mui-org/material-ui-x/issues/196) if you want to see it land faster.
+## Translation keys
+
+You can use the `localeText` prop to pass in your own text and translations.
+You can find all the translation keys supported in [the source](https://github.com/mui-org/material-ui-x/blob/HEAD/packages/grid/_modules_/grid/constants/i18nConstants.ts) in the GitHub repository.
```

Comment:
We should perhaps document them properly at some point.

Reply:
`localeText` prop is added to the api pages of both DataGrid and xGrid. If you are talking about the constant, I've exported an interface with the typing so that people can check it from the code.

Reply:
> We should perhaps document them properly at some point.

@mbrookes We don't do it either for the core components. I have seen ag-Grid render the default value for the locale. But linking the page on GitHub with the correct version could be as good.